src/App/App.jsx

```diff
@@ -5,7 +5,7 @@ import "@fortawesome/fontawesome-free/css/all.css";
 import "./index.css";
 
 // Import configurations and utilities
-import { config, SETTINGS } from "../config/main";
+import { CONFIG, SETTINGS } from "../config/index";
```

Comment:
So a very cool thing about the `index.js` file is that it will automatically get resolved in import statements! So you can actually do this for all of them:
```suggestion
import { CONFIG, SETTINGS } from "../config/";
```